commands/template_store_list.go

```diff
@@ -0,0 +1,175 @@
+package commands
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"text/tabwriter"
+
+	"github.com/spf13/cobra"
+)
+
+const (
+	// The URL where the store resides
+	DefaultTemplateList = "https://raw.githubusercontent.com/martindekov/store/martindekov/unofficial_templates_json/templates.json"
+	incubatorRepo       = "openfaas-incubator"
+	officialRepo        = "openfaas"
+	allTemplates        = "all"
+)
+
+var (
+	source string
+)
+
+func init() {
+	templateStoreListCmd.Flags().StringVarP(&source, "source", "s", allTemplates, "The source of the templates")
+	templateStoreListCmd.Flags().BoolVarP(&verbose, "verbose", "v", false, "Shows additional language and platform")
+
+	templateStoreCmd.AddCommand(templateStoreListCmd)
+}
+
+var templateStoreListCmd = &cobra.Command{
+	Use:     `list`,
+	Short:   `List templates from OpenFaaS organizations`,
+	Aliases: []string{"ls"},
+	Long:    `List templates supported by openfaas or openfaas-incubator organizations or both`,
```

Comment:
How do I specify the store url like with the function store commands?

Reply:
Is this a miss?

Reply:
I added `--url` flag for both `pull` and `list` commands ✔️